x/exercise.rb

```diff
@@ -0,0 +1,7 @@
+module X
+  module Exercise
+    def self.exists?(language, slug)
+      Xapi.request('tracks', language, slug).status != 404
```

Comment:
@kytrinyx Can actually talking to the endpoint be avoided in this case? I think we can mock `Xapi.request` here (just checking that the method was called with proper args). We'll then need to write test for [Xapi](https://github.com/exercism/exercism.io/blob/master/x/xapi.rb) where we mock `Faraday` and trust it to make the proper request.